synchronizer/l1_syncstatus.go

```diff
@@ -92,6 +92,17 @@ func (s *syncStatus) isNodeFullySynchronizedWithL1() bool {
 	return false
 }
 
+func (s *syncStatus) getNextRangeOnlyRetries() *blockRange {
+	s.mutex.Lock()
+	defer s.mutex.Unlock()
+	// Check if there are any range that need to be retried
+	blockRangeToRetry, err := s.errorRanges.getFirstBlockRange()
```

Comment:
Why this function doesn't use the Unsafe?

Reply:
Yes, you're right!, I'm duplicating code. I'll fix it